### DISCUSSION TITLE
"Edit this page" button is visible locally, but missing when published

### DISCUSSION THREAD

**Discussion by SerVB** [squidfunk/mkdocs-material#4916] [Q&A]
Hi! I have a question: how can I locate the problem?

When I launch `mkdocs serve` locally (or do `mkdocs build` and open the resulting `index.html` in my browser), I see the "edit this page" pencil button on my docs pages. However, when the docs are built on CI (`mkdocs build`) and the files are served via a hosting, the pencil buttons are missing.

I haven't noticed other differences. Link to my Git repo works well in both places, as well as searching.

I can't share the docs fully because they are internal. But if some pieces are needed, I of course can consider sharing them.

Thanks in advance!

**Comment by squidfunk** [comment]
My first idea: sounds like a caching problem. Make sure that the latest version of `mkdocs-material` is installed in your CI. If that doesn't work, the best idea is to craft a [small reproduction](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/) that we can look into, reducing your case down to the absolute minimum.
